docs/topics/appendix/settings.rst

```diff
@@ -66,3 +66,7 @@ Settings
   * If this is set, it will be read as a number of
     bytes.  Any messages above that size will not be
     downloaded.  ``2000000`` is 2 Megabytes.
+* ``DJANGO_MAILBOX_STORE_ORIGINAL_MESSAGE``
+  * Default: ``False``
```

Comment:
Thank you so much for updating this documentation! :heart:

`rst` files are fairly picky, and if you don't put a newline above this (70) and above the previous line (69), it won't be properly formatted (see https://github.com/ad-m/django-mailbox/blob/master/docs/topics/appendix/settings.rst).